scipy/special/_multiufunc.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+class multiufunc:
```

Comment:
somebody else mentioned it elsewhere, but [PEP8 says](https://peps.python.org/pep-0008/#class-names) to style this as `MultiUfunc` (or `Multiufunc`, if it's supposed to be one word)

Reply:
Indeed, thanks @lucascolley! I have renamed `multiufunc` to `MultiUFunc`.